example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -625,22 +625,22 @@ describe('When there are granule differences and granule reconciliation is run',
       reportRecord = JSON.parse(asyncOperation.output);
     });
 
-    it('Fetches a signed URL to the Granule Inventory report through the Cumulus API', async () => {
+    it('Fetches a object with a signedURL to the Granule Inventory report through the Cumulus API', async () => {
```

Comment:
Small nitpick, but this should say "an" object

Reply:
I will sneak this into #1920